replication-api/src/main/java/com/connexta/replication/api/impl/persistence/ReplicationItemManager.java

```diff
@@ -43,21 +43,14 @@
    * @return list of items for the given {@link ReplicatorConfig} id
    * @throws ReplicationPersistenceException if there is an error fetching the items
    */
-  List<ReplicationItem> getItemsForConfig(String configId, int startIndex, int pageSize);
-
-  /**
-   * Saves a new {@code ReplicationItem}.
-   *
-   * @param replicationItem the item to save.
-   */
-  void saveItem(ReplicationItem replicationItem);
+  List<ReplicationItem> getAllForConfig(String configId, int startIndex, int pageSize);
 
   /**
    * Deletes all {@code ReplicationItem}s.
    *
    * @throws ReplicationPersistenceException RuntimeException thrown if delete was unsuccessful
    */
-  void deleteAllItems();
+  void removeAll();
```

Comment:
❓ Do we use this anywhere? Can we delete it? This was a left over from the config remove command for easy cleanup.

Reply:
I am all for removing it.